x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_menu.tsx

```diff
@@ -75,7 +79,10 @@ export const ActionTypeMenu = ({
   }, []);
 
   const registeredActionTypes = Object.entries(actionTypesIndex ?? [])
-    .filter(([id, details]) => actionTypeRegistry.has(id) && details.enabledInConfig === true)
+    .filter(
```

Comment:
instead of filter out a specific id maybe we could create something like `groupKey` and set it to `.slack` for both connector types and filter by whether the group key already exists in the registeredActionTypes?

Reply:
we are going to add a new prop called `hideFromConnectorList` to be set to true in those connector types that are subtypes and should not be shown in the connector list.